templates/patient/card/card_base.html.twig

```diff
@@ -1,7 +1,7 @@
 <section class="{{ card_container_class_list|default(['card', 'mb-2'])|join(' ') }} {{ card_bg_color }} {{ card_text_color }}">
     <div class="card-body p-1">
         <h6 class="card-title mb-0 d-flex p-1 justify-content-between">
-            <button class="btn btn-sm btn-link" href="#" {% if forceAlwaysOpen != true %} data-toggle="collapse" data-target="#{{ id|attr }}" {% endif %} aria-expanded="true" aria-controls="{{ id|attr }}">{{ title|text }}</button>
+            <a class="text-left font-weight-bolder" href="#" {% if forceAlwaysOpen != true %} data-toggle="collapse" data-target="#{{ id|attr }}" {% endif %} aria-expanded="true" aria-controls="{{ id|attr }}">{{ title|text }}<i class="ml-1 fa fa-fw fa-sm fa-expand" data-target="#{{ id|attr }}"></i></a>
```

Comment:
The font-weight-bolder will fix our title issue! Though don't know why not jus font-weight-bold but whomever styled may have a reason.